test/test.cpp

```diff
@@ -0,0 +1,88 @@
+#include <loudnessIO/SoundFile.hpp>
+#include <loudnessIO/ProcessFile.hpp>
+#include <loudnessAnalyser/LoudnessAnalyser.hpp>
+
+#include "gtest/gtest.h"
+
+#include <string>
+#include <cmath>
+#include <stdlib.h>
+#include <iostream>
+
+// Value of EBU_LOUDNESS_TEST_SET_PATH environment variable
+std::string ebuLoudnessTestSetPath;
+
+/**
+ * @brief Callback used while processing, to print progression of the loudness analysis.
+ */
+void progress(const int p)
+{
+    std::cout << "[" << p << "%]\r" << std::flush;
```

Comment:
I would prefer to remove this function, since I do not care of progress within the unit tests. But the ___Ploud___ API does not allow any process without a callback function.